conf/app.ini

```diff
@@ -2,36 +2,37 @@
 APP_NAME = Gitea
 RUN_USER = __APP__
 RUN_MODE = prod
+WORK_PATH =__INSTALL_DIR__
 
 [actions]
 ENABLED = true
```

Comment:
```suggestion
ENABLED = false
```

Reply:
I would rather set actions support as false. (https://forum.yunohost.org/t/has-anyone-managed-to-get-gitea-actions-working-with-yunohost/27064)